business-central-parent/business-central-home-page-product/src/main/java/org/kie/bc/client/navbar/AboutPopupConfigImpl.java

```diff
@@ -73,7 +73,7 @@ public String productImageUrl() {
 
     @Override
     public String backgroundImageUrl() {
-        return "images/product_home_bg.png";
```

Comment:
@RishiRajAnand I think you could also remove the product_home_bg.png file from the project since looks like it is no longer used. However consider this as optional ;)

Reply:
Sure, I'll do it

Reply:
Done, Its better to delete it so as to minimize the build size. Thanks for the observation @pefernan